pkgs/by-name/ve/vencord/package.nix

```diff
@@ -36,20 +35,6 @@ stdenv.mkDerivation (finalAttrs: {
   ];
 
   env = {
-    ESBUILD_BINARY_PATH = lib.getExe (
-      esbuild.overrideAttrs (
-        final: _: {
-          version = "0.25.1";
-          src = fetchFromGitHub {
-            owner = "evanw";
-            repo = "esbuild";
-            rev = "v${final.version}";
-            hash = "sha256-vrhtdrvrcC3dQoJM6hWq6wrGJLSiVww/CNPlL1N5kQ8=";
-          };
-          vendorHash = "sha256-+BfxCyg0KkDQpHt/wycy/8CTG6YBA/VJvJFhhzUnSiQ=";
-        }
-      )
-    );
```

Comment:
Could you document why you removed this in the commit for future reference? Both `pkgsCross.aarch64-multiplatform.vencord` on x86_64-linux and `vencord` on aarch64-linux build fine without the devendored esbuild, so the change seems okay

Reply:
I honestly don't know why it was there in the first place and it seemed unnecessary so I gave it a try without it, which worked.

Reply:
> I honestly don't know why it was there in the first place and it seemed unnecessary so I gave it a try without it, which worked.

Unvendored esbuilds are usually needed, if the upstream lockfile does not have locks for esbuild binaries for all platforms we need. A lot of projects only include binaries for x86_64-linux and some darwin platforms. I guess something in the dependency tree of vencord has fixed this, which is why it's seemingly not needed anymore

Reply:
I added a comment to my commit message